src/hokusai/hokusai.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         default: deploy
       filename:
         type: string
-        default: ./hokusai/test.yml
+        default: ./hokusai/test.yml.j2
```

Comment:
```suggestion
```
maybe lets just remove this totally, and let hokusai do its thing guessing the right file, but keeping the option to specify a filename.